junit-platform-engine/src/main/java/org/junit/platform/engine/reporting/ReportEntry.java

```diff
@@ -35,6 +36,14 @@
 	private final LocalDateTime timestamp = LocalDateTime.now();
 	private final Map<String, String> keyValuePairs = new LinkedHashMap<>();
 
+	/**
+	 * @deprecated Use {@link #from(String, String)} or {@link #from(Map)}
+	 */
+	@API(status = DEPRECATED, since = "5.8")
+	@Deprecated
+	public ReportEntry() {
+	}
+
```

Comment:
We didn't mean for the default constructor to be public, did we?

Reply:
I do not recall properly, whether this was intentional.
In hindsight, it seems to be an accident rather than explicitly designed this way.